extension/src/plots/model/index.ts

```diff
@@ -307,6 +313,26 @@ export class PlotsModel extends ModelWithPersistence {
     this.persist(PersistenceKey.PLOT_COMPARISON_ORDER, this.comparisonOrder)
   }
 
+  public setComparisonMultiPlotValue(
+    revision: string,
+    path: string,
+    value: number
+  ) {
+    if (!this.comparisonMultiPlotValues[revision]) {
+      this.comparisonMultiPlotValues[revision] = {}
```

Comment:
Should we look into removing old revisions from the `comparisonMultiPlotValues` state? Updating the state every time a revision gets removed, for example?

Reply:
If we were going to do that I would check to see if the revision is in the full list of experiments/commits before removing